llvm/docs/DirectX/DXContainer.rst

```diff
@@ -111,7 +111,7 @@ FXC are marked with \*.
 #. `PSV0`_ - Stores Pipeline State Validation data.
 #. RDAT† - Stores Runtime Data.
 #. RDEF\* - Stores resource definitions.
-#. RTS0 - Stores compiled root signature.
+#. `RTS0`_ - Stores compiled root signature.
 #. `SFI0`_ - Stores shader feature flags.
```

Comment:
It's odd that the order here doesn't match the order of the sections later. Probably makes sense to insert the new content right before the SFI0 section rather than right after.